packages/marginfi-client-v2/src/client.ts

```diff
@@ -187,9 +187,9 @@ class MarginfiClient {
         if (response.status === 200) {
           const parsedResponse = await response.json();
           if (!parsedResponse) throw new Error("JSON is mia");
-          const loopupTableString = parsedResponse[config.groupPk.toString()];
+          const lookupTableString = parsedResponse[config.groupPk.toString()];
           if (!parsedResponse) throw new Error("Group not found");
-          addressLookupTableAddresses = [new PublicKey(loopupTableString)];
+          addressLookupTableAddresses = [new PublicKey(lookupTableString)];
```

Comment:
@masterzorgon has this been tested? just want to ensure that variable is not used anywhere else. as this is in the main sdk so high risk.